plugins/inventory/inventory.py

```diff
@@ -204,6 +204,22 @@
       description: List of custom ansible host vars to create from the device object fetched from Nautobot
       default: {}
       type: dict
+    rename_variables:
+      description:
+          - Rename variables evaluated by nb_inventory, before writing them.
+          - Each list entry contains a dict with a 'pattern' and a 'repl'.
+          - Both 'pattern' and 'repl' are regular expressions.
+          - You can use rename_variables to change variable names before the inventory gets loaded.
+          - rename_variables:
+          - `  - pattern: "cluster"`
```

Comment:
This is failing an ansible sanity test:

> DOCUMENTATION: syntax error: found character '`' that cannot start any token (syntax)

You can't use backticks in ansible documentation. Please move this section to the bottom of the EXAMPLES variable/section (not here in the DOCUMENTATION variable) as I requested, instead.